### ISSUE TITLE
BIOS support

### ISSUE THREAD

**Issue by MasterMegatron** [shalxmva/modxo#25]
Hi! Does this chip support different bios besides cerbios? I want to make a 128mb ram mod on my xbox 1.6, and for this I need xblast bios for ram tests. For 1.6 there is a modified xblast. Now I only have a softmod, and I’m still thinking about which modchip to choose is best. Thanx in advanse.

**Comment by shalxmva** [comment]
Yes, i have run Evox. Try using the packer utility (preferably the python script at the repository).

**Comment by shalxmva** [comment]
Please feel free to test any other Bios and notify if there is  one that doesn't work
